docs_src/src/pages/documentation/example_app/templates.mdx

```diff
@@ -65,6 +65,63 @@ This is Robyn introduced him to the concept of routers and views.
   />
 </div>
 
+# Jinja Integration Instructions
```

Comment:
@carrycooldude , the jinja integration docs are already present in the file. you need to add a specific documentation for `url_for`